cloud_governance/common/jira/jira_operations.py

```diff
@@ -190,6 +191,20 @@ def get_issue_sub_tasks_cost_estimation(self, ticket_id: str):
             cost_estimation += float(description.get('CostEstimation', 0))
         return cost_estimation
 
+    @typeguard.typechecked
+    def get_issue_sub_tasks_duration(self, ticket_id: str):
+        """
+        This method return the issue sub-tasks total duration
+        :param ticket_id:
+        :return:
+        """
+        sub_tasks = self.get_ticket_id_sub_tasks(ticket_id=ticket_id, closed=True)
+        total_duration = 0
+        for sub_task in sub_tasks:
+            description = self.get_issue_description(ticket_id=sub_task, sub_task=True)
+            total_duration += int(description.get('Days', 0))
```

Comment:
How do you know that its the extend duration, is it different parameter or same ?
I ask it because you can take the original duration and not the extend.

Reply:
if see the for loop. I am getting this from sub_tasks. All subtasks are for extension.